compiler/damlc/tests/daml-test-files/InternedExternalRefs.daml

```diff
@@ -2,7 +2,14 @@
 -- All rights reserved.
 
 -- @ SINCE-LF 1.6
-daml 1.2 module InternedExternalRefs where
+-- @QUERY-LF [ .modules[] | .values[] | .expr.val.module.package_ref.interned_id ] | sort == ["0", "1"]
```

Comment:
I assume this test would have worked before as well?

Reply:
Yes, it did. The old test didn't make me very confident since `0` is the default for numbers and hence `>= 0` might be true even if some number isn't actually present. I hope this is more robust.